### ISSUE TITLE
Suggestion for vpn locations.

### ISSUE THREAD

**Issue by Disposable-account** [ivpn/desktop-app#320]
# Feature request
Possibly add 2 servers in the empty spaces of Africa,1-2 more in south America, and one in the space between Holon Tel Aviv to Hong Kong. Or seeing of servers close to censored countries.
## Description
Idk if im suppose to say this here and this is kinda the 1st time I am trying doing this in general, though it seems that some places are empty that could have a server like the middle part of Africa and higher left of it so people in these places have closer+faster connections. This is the same for the spot between Holon Tel Aviv and Hong Kong which Nepal could work but I am unsure of there laws. For south America, there could be 1-2 more servers in Argentina or Colombia since Brazil is the only one in it. I am unsure about servers in Russia+China though it is likely not possible without logging or other stuff. So for those people a suggestion is adding servers close to those countries like Mongolia, Kazakhstan, if those work. Miscellaneous suggestions is Estonia I think have good privacy/vpn laws and is close to Russia and possibly 1 more vpn server held in countries at war like Israel and Ukraine in the case one of the servers are destroyed or cancelled from war like abandoned because of evacuation or shut down to prevent capture or there were constant problems from war.
I will mention I don't know the laws and stuff for most of these countries nor have time to find them so idk if some of these have logging laws or if they are valid for Ivpn to have. I also don't expect all or most suggestions to be done because of budget or if there are more important/strategical locations to do.

## Describe the solution you'd like

TLDR is suggestions in feature request and censored places  though this is very simplified.
## Describe alternatives you've considered

Have you considered any alternative solutions or workarounds?

**Comment by jurajhilje** [comment]
Hi, thanks for the feedback and suggestions. I will relay this to the team and close the issue as it is not related to our Desktop app.
For general questions and feedback, I suggest our Reddit channel - https://www.reddit.com/r/IVPN/
